legacy/skills/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,8 @@
 <resources>
   <!-- Main style guide colors -->
   <color name="styleguide_pink">#FF6381</color>
-  <color name="styleguide_blue">#111B39</color>
+  <color name="styleguide_blue">#1A1A24</color>
+  <color name="styleguide_blue_secondary">#242333</color>
   <color name="styleguide_medium_grey">#C9C9C9</color>
```

Comment:
it need to be changed later, I opened another PR with the correct local for that

Reply:
ok, in that case it's better if your PR goes first to the feature-branch. I will merge it with mine after, and apply the new changes to mine